lib/blocs/simple_bloc_observer.dart

```diff
@@ -14,8 +14,8 @@ class SimpleBlocObserver extends BlocObserver {
   }
 
   @override
-  void onError(Cubit cubit, Object error, StackTrace stackTrace) {
-    print('onError $error');
-    super.onError(cubit, error, stackTrace);
+  void onError(BlocBase bloc, Object error, StackTrace stackTrace) {
```

Comment:
Is this for better stacktraces?

Reply:
Should be. Its the one used in the bloc-documentation for BlocObservers : https://pub.dev/documentation/bloc/latest/

Reply:
It is also worth mentioning, that this crashes on the newest version of bloc_test (8.0.0). I'm guessing its due to bloc_test implementing a "MockCubit", which means that that function no longer can cast to Bloc, as they are now two seperate functions.

Reply:
> A lot of spread out comments here.
> 
> -The obvious dev-code will be removed. This is being worked on as we speak
> -The tests have not yet been implemented. "CreateBlocProviderForTesting" is legacy and will be removed.
> 
> * The routing is also being worked on. Coming in its own PR.

Awesome, will approve it. Merge when you feel ready! :)